README.md

```diff
@@ -48,7 +53,10 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+town[:resident] - ["Belle"]
```

Comment:
Line 57 should be `belle = town[:residents].delete "Belle" `instead for this question.` - `is not a valid character in ruby. Line 58 solution is great!